docs/guide/roles-and-multi-sig.md

```diff
@@ -0,0 +1,261 @@
+
+```
+{
+  "height": 81,
+  "data": {
+    "coins": [
+      {
+        "denom": "mycoin",
+        "amount": 9007199254740992
+      }
+    ],
+    "credit": []
+  }
+}
+```
+
+whereas `poor` and `igor` have no coins (in fact, the chain doesn't know about them yet):
+
+```
+ERROR: Account bytes are empty for address 65D406E028319289A0706E294F3B764F44EBA3CF
+```
+
+## Create Role
+
+This first step defines the parameters of a new role, which will have control of any coins sent to it, and only release them if correct conditions are met. In this example, we are going to make a 2/3 multi-sig wallet. Let's look a the command and dissect it below:
+
+```
+basecli tx create-role --role="10CAFE4E" --min-sigs=2 --members=5E4CB7A4E729BA0A8B18DE99E21409B6D706D0F1,65D406E028319289A0706E294F3B764F44EBA3CF,CB76F4092D1B13475272B36585EBD15D22A2848D --sequence=1 --name=rich
+```
+
+In the first part we are sending a transaction that creates a role, rather than transfering coins. The `--role` flag is the name of the role (in hex only) and must be in double quotes. The `--min-sigs` and `--members` define your multi-sig parameters. Here, we require a minimum of 2 signatures out of 3 members but we could easily say 3 of 5 or 9 of 10, or whatever your application requires. The `--members` flag requires a comma-seperated list of addresses that will be signatories on the role. Then we set the `--sequence` number for the transaction, which will start at 1 and must be incremented by 1 for every transaction from an account. Finally, we use the name of the key/account that will be used to create the role, in this case the account `rich`. 
+
+Remember that `rich`'s address was used on `basecoin init` and is included in the `--members` list. The command above will prompt for a password (which can also be piped into the command if desired) then - if executed correctly - return some data:
+
+```
+{
+  "check_tx": {
+    "code": 0,
+    "data": "",
+    "log": ""
+  },
+  "deliver_tx": {
+    "code": 0,
+    "data": "",
+    "log": ""
+  },
+  "hash": "4849DA762E19CE599460B9882DD42C7F19655DC1",
+  "height": 321
+}
+```
+showing the block height at which the transaction was committed and its hash. A quick review of what we did: 1) created a role, essentially an account, that requires a minimum of two (2) signatures from three (3) accounts (members). And since it was the account named `rich`'s first transaction, the sequence was set to 1.
+
+Let's look at the balance of the role that we've created:
+
+```
+basecli query account role:"10CAFE4E"
+```
+
+and it should be empty:
+
+```
+ERROR: Account bytes are empty for address role:10CAFE4E
+```
+
+Next, we want to send coins _to_ that role. Notice that because this is the second transaction being sent by rich, we need to increase `--sequence` to `2`: 
+
+```
+basecli tx send --fee=90mycoin --amount=10000mycoin --to=role:"10CAFE4E" --sequence=2 --name=rich
```

Comment:
copied from the tests: https://github.com/tendermint/basecoin/blob/unstable/tests/cli/roles.sh#L10 and https://github.com/tendermint/basecoin/blob/unstable/tests/cli/roles.sh#L10